plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/inner-blocks/checkbox-list/style.scss

```
@@ -36,6 +36,7 @@
 	}
 }
 
+span.wc-block-product-filter-checkbox-list__input-wrapper
 :where(.wc-block-product-filter-checkbox-list__input) {
```

In this case, I think we can remove `:where` and use normal selector.
```suggestion
input.wc-block-product-filter-checkbox-list__input {
```